src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -1604,6 +1604,10 @@ public String getDisplayName(RendererConfiguration mediaRenderer) {
 		return getDisplayName(mediaRenderer, true);
 	}
 
+	public boolean isInsideTranscodeFolder() {
+		return parent != null && parent instanceof FileTranscodeVirtualFolder;
```

Comment:
`parent instanceof FileTranscodeVirtualFolder` also checks that it's non-null. ´null´ is never the instance of anything.

Reply:
Thanks, I removed that check